apps/website/.vuepress/theme/global-components/ItemOverview.vue

```diff
@@ -0,0 +1,48 @@
+<template>
+  <section cds-layout="vertical" role="link" @click="navigate" class="item-overview">
+    <h2 cds-text="section">
+      <div cds-layout="horizontal gap:md p:lg align:center">
+        <a :href="path" :target="target"> {{ title }} </a>
+        <span role="status" v-if="isNew" class="badge badge-info">new</span>
+      </div>
+    </h2>
+    <div cds-layout="align:center p:md">
+      <slot></slot>
+    </div>
+    <slot name="footer"></slot>
+  </section>
+</template>
+
+<script>
+export default {
+  name: 'ItemOverview',
+  props: ['title', 'path', 'target', 'isNew'],
+  methods: {
+    navigate: function () {
+      window.location = this.path;
```

Comment:
Can we do it with Vue router here?

Reply:
The Vue router is available in the theme components. I think it is provided and accessed via `this.$route`.
@shijir didn't block so I'm assuming he considers it a nice to have and not a blocking issue to these changes.